### PR TITLE
Add License

### PR DESCRIPTION
**Pull request by sebpuetz**

### CODE REVIEW

**Comment by sebpuetz** [comment]
Seems like I forgot to request the review, oops

File: CONTRIBUTORS [hunk 1]
```diff
@@ -0,0 +1,11 @@
+Copyright 2019 The finalfusion-tensorflow contributors
+
+finalfusion-tensorflow is provided to you by its contributors under the Blue Oak
+Model License Version 1.0.0 (see LICENSE.md). Both Daniël de Kok and Sebastian
```
Comment (danieldk) [inline CONTRIBUTORS]: I think you did the initial work with to op with Tobias? So, it may be fairer to put Tobias in place of me.

**Review by danieldk — APPROVED** [review]

File: CONTRIBUTORS (hunk 1 above)
Comment (sebpuetz) [reply CONTRIBUTORS]: I added the three of us if that's fine with you

**Comment by sebpuetz** [comment]
@twuebi

**Review by twuebi — APPROVED** [review]

**Pull request merged**